Makefile

```diff
@@ -0,0 +1,24 @@
+INCLUDES= checkers
+
+
+# Builds the ppx rewriter names sample
+build:
+	ocamlbuild -Is $(INCLUDES) -package compiler-libs.common linter.native
+
+
+# Runs the ppx rewriter on test code
+test:
+	ocamlfind ppx_tools/rewriter ./linter.native test.ml
```

Comment:
Might be good to change this to refer to a folder full of test files, but this is fine for now.

Reply:
Definitely - I'll make this change at a later date, but for the sake of simplifying the development process I'm going to leave this as is.